test/tripRequest.test.js

```diff
@@ -204,6 +204,91 @@ describe('perform CRUD operations on trip request', () => {
       });
   });
 
+  /* it('should get trips status of year, month and day', (done) => {
```

Comment:
remove this comments

Reply:
those tests are needed, since  the trip request can't approve themselves those tests can't work, so they are waiting for the `approve/disapprove PR` to be merge then they can be uncomment.

Reply:
copy that